src/components/v5/common/ActionSidebar/partials/motions/MotionSimplePayment/MotionSimplePayment.tsx

```diff
@@ -97,12 +98,28 @@ const MotionSimplePayment: FC<MotionSimplePaymentProps> = ({
             },
             {
               key: MotionState.Reveal,
-              content: <RevealStep />,
+              content: (
+                <RevealStep
+                  motionData={motionData as ColonyMotion}
+                  startPollingAction={startPollingForAction}
+                  stopPollingAction={stopPollingForAction}
+                  transactionId={transactionId}
+                />
+              ),
               heading: {
                 label: formatText({ id: 'motion.reveal.label' }) || '',
+                decor:
+                  motionStateEnum === MotionState.Reveal && motionStakes ? (
```

Comment:
Can't it just be like this:  motionStateEnum === MotionState.Reveal && motionStakes && ( ... ) ?

Reply:
Decor is types as optional JSX.Element so it needs undefined if there is no content